src/app/components/markdown-control/markdown-control.component.less

```diff
@@ -65,26 +65,26 @@ pre { margin: 0; }
         li {
           &.active {
             a {
-              border-bottom-color: $color-pf-white;
-              @include Border(1px, solid, $color-pf-blue-400);
+              border-bottom-color: @color-pf-white;
+              .Border(1px, solid, @color-pf-blue-400);
```

Comment:
why? it has to be set each time, and the way the mixin is written, makes it more difficult to find errors or changes.